### PR TITLE
[WIP] Apigee OpenLegacy Kickstart

### PR DESCRIPTION
**Pull request by laughingbiscuit**
*Status*
Work in progress

*Description*
A quickstart script to deploy OpenLegacy to Google Cloud Functions, Generate and deploy an Apigee Proxy and run some tests.

*Backlog*
 - [x] Build skeleton of code 
 - [x] Build skeleton of docs
 - [x] Spike OpenLegacy Docker integration
 - [x] Spike OpenAPI spec to AssignMessage policy 
 - [x] Finalise code
 - [x] Finalise docs
 - [x] Create pipeline
 - [ ] Apigee Review
 - [x] OpenLegacy Review 

---

- [ ] I have run all the tests locally and they all pass.
- [ ] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Comment by laughingbiscuit** [comment]
will review with @seymen tomorrow and add require secrets before running gcbrun

File: tools/apigee-openlegacy/bin/apigee-openlegacy.sh [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+docker build -t gcr.io/"$GCP_PROJECT"/aok-image:latest .
+docker push gcr.io/"$GCP_PROJECT"/aok-image:latest
+
+
+###
+# Deploy OpenLegacy
+###
+
+gcloud run deploy aok-service --image gcr.io/"$GCP_PROJECT"/aok-image:latest --platform managed --region europe-west1 -q
```
Comment (danistrebel) [inline tools/apigee-openlegacy/bin/apigee-openlegacy.sh:96]: Parametrize region?

Comment (laughingbiscuit) [reply tools/apigee-openlegacy/bin/apigee-openlegacy.sh]: done - thanks

**Review by seymen — APPROVED** [review]

**Pull request merged**